Assets/Shopify/UIToolkit/Editor/ProductPicker.cs

```diff
@@ -20,10 +20,10 @@ public interface IProductPickerController {
     }
 
     public class ProductPicker : IProductPickerController {
+        public ShopifyClient Client;
```

Comment:
Made this public so we can change it if it updates

Reply:
Hmmm... Is this weird to be able to modify the client?